config.json

```diff
@@ -26,6 +26,37 @@
     "practice": []
   },
   "concepts": [],
-  "key_features": [],
+  "key_features": [
+    {
+      "title": "Strong, dynamic typing",
+      "content": "Gdscript is strong and dynamically typed and supports gradual typing.",
+      "icon": "dynamically-typed"
+    },
+    {
+      "title": "Cross platform",
+      "content": "Gdscript code can run on Windows, Linux, macOS, iOS, Android, and the web.",
+      "icon": "cross-platform"
+    },
+    {
+      "title": "C++ bindings",
+      "content": "C++ bindings allows for fast, low-level access to the engine's features.",
+      "icon": "fast"
+    },
+    {
+      "title": "Specialized types for game development",
+      "content": "Specialized types for increased productivity and performance, when working with 2D and 3D games.",
+      "icon": "powerful"
+    },
+    {
+      "title": "Garbage collection",
+      "content": "Garbage collection is automatic and seamless, freeing you from manual memory management.",
+      "icon": "garbage-collected"
+    },
```

Comment:
Godot is using reference counting for memory management, not garbage collection. It's one of the features of GDScript, _not being_ garbage collected. :)

Reply:
```suggestion
    {
      "title": "No garbage collection",
      "content": "Garbage collection can cause random freezes during gameplay. Godot (and thus GDScript) instead use reference counting, freeing you from manual memory management.",
      "icon": "garbage-collected"
    },
``